### PR TITLE
Added test to reproduce prefixed length aspect.

### PR DESCRIPTION
**Pull request by mbeckerle**
DAFFODIL-2626

### CODE REVIEW

**Review by stevedlawrence — APPROVED** [review]
+1

File: daffodil-test/src/test/scala/org/apache/daffodil/section17/calc_value_properties/TestComputedLengthFields.scala [hunk 1]
```diff
@@ -43,4 +43,9 @@ class TestComputedLengthFields {
 
   @Test def test_computedLengthAroundFixedLengths1(): Unit = { runner.runOneTest("computedLengthAroundFixedLengths1") }
 
+  // DAFFODIL-2626 circular deadlock
+  // Reproduces one of the circular issues - with prefixed length for the root element surrounding
+  // text, where the alignment region
```
Comment (tuxji) [inline daffodil-test/src/test/scala/org/apache/daffodil/section17/calc_value_properties/TestComputedLengthFields.scala:48]: Comment ends with an incomplete phrase - did you mean "where the alignment region isn't optimized out"?

**Review by tuxji — APPROVED** [review]
+1

File: daffodil-test/src/test/scala/org/apache/daffodil/section17/calc_value_properties/TestComputedLengthFields.scala (hunk 1 above)
Comment (mbeckerle) [reply daffodil-test/src/test/scala/org/apache/daffodil/section17/calc_value_properties/TestComputedLengthFields.scala:48]: oooo. I did that on purpose just to keep you guys all on your toes :-)

**Pull request merged**